src/components/game/gameHelper.js

```diff
@@ -0,0 +1,31 @@
+import * as cellValue from '../../cellValue';
+import * as gameStatus from '../../gameStatus';
+import * as lineAnalysis from '../../lineAnalysis';
+
+export function determineMark(grid) {
+  let numberOfX = countMark(cellValue.X, grid);
+  let numberOfO = countMark(cellValue.O, grid);
+  return (numberOfX > numberOfO) ? cellValue.O : cellValue.X
+}
+
+export function countMark(mark, grid) {
+  let count = 0;
+  let currentGrid = grid;
+  currentGrid.forEach(function(cell) {
+    if(cell === mark)
+      count++;
+  });
+  return count;
+}
+
+export function status(grid) {
```

Comment:
Is there a more descriptive name than `status` for this? I always start my function names off with a verb.